html/index.html

```diff
@@ -1,30 +1,30 @@
 <!DOCTYPE html>
 <html>
 <head>
-	<title>Test Webpage</title>
+    <title>Test Webpage</title>
```

Comment:
The entire file should be indented using 4 spaces, not tabs, and have a trailing newline.